Cleanup README (add pact-lsp, remove stack instructions) (#1225)
kadena-io/pact: README.md

## Supported Editors

Pact is supported by a variety of editors ranging from full-fledged IDE environments to syntax highlighting.

> Moreover, our offering includes a Pact Language Server, which can be seamlessly integrated with a wide range of editors.

Reviewer: ```suggestion
Moreover, we also provide a Pact Language Server, which can be seamlessly integrated with a wide range of editors.
```
